app/views/devise/registrations/index.html.erb

```diff
@@ -1,7 +1,10 @@
 <div class="container-fluid" style="padding-top: 20px; padding-left: 30px">
+  <h1>Controls</h1>
+  <h4> Donate page visibility toggle </h4>
+    <%= button_to "Toggle", {action: "toggle_donation_show"}, {class: "btn btn-primary"} %>

```

Comment:
I think it would be more intuitive/clear for them if it was a checkbox like, "donation page disabled [x]" or something similar like a switch, instead of "Toggle". It makes sense to me, and looks good, I just feel like maybe for them it would be clearer and neater maybe?

Reply:
I'm not really sure how to do it with a checkbox - I agree it would be more intuitive, but there would still be a button that submits the "form" to trigger the toggle_donation_show method and reloads the page. Doing it with a checkbox would require writing javascript that hides it on your screen first, and still clicking a button to actually trigger the change. In general I think this isn't something rails is well equipped to handle because usually master would just not have the donation code at all, and the webmaster would choose when to add it back. This seems like the best option considering we won't be around to manually hide and unhide the donation.